website/docs/_advanced_install.mdx

```diff
@@ -363,6 +363,17 @@ Script to automatically download and cache standalone `scala-cli` launcher.
 </Tabs>
 </div></div>
 
+<SectionAbout title="Bootstrapped standalone fat JAR" colBigTitle="7"></SectionAbout>
+
+<div className="row"><div className="col col--9 col--offset-1 padding--lg advanced_install_methods">
+
+To run the bootstrapped Scala CLI standalone fat jar using Coursier, follow the command below:
+
+```bash
+cs launch org.virtuslab.scala-cli:cliBootstrapped:latest.release -M scala.cli.ScalaCli
+```
```

Comment:
it may also be worth it to separately link the URL to to the maven repo: https://repo1.maven.org/maven2/org/virtuslab/scala-cli/cliBootstrapped